DigitalLearningSolutions.Data.Tests/DataServices/GroupsDataServiceTests.cs

```diff
@@ -402,19 +402,68 @@ public async Task DeleteGroupDelegates_deletes_all_group_delegates()
         }
 
         [Test]
-        public async Task DeleteGroupCustomisations_deletes_all_group_customisations()
+        public void DeleteGroup_deletes_group()
         {
-            using var transaction = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled);
+            using var transaction = new TransactionScope();
             try
             {
                 // Given
-                const int groupId = 8;
+                const int groupId = 25;
+                const int centreId = 101;
 
+                // When
+                groupsDataService.DeleteGroup(groupId);
+
+                // Then
+                var groupName = groupsDataService.GetGroupName(groupId, centreId);
+                groupName.Should().BeNull();
+            }
+            finally
+            {
+                transaction.Dispose();
+            }
+        }
+
+        [Test]
+        public void GetGroupCourse_returns_expected_course()
+        {
+            // Given
+            var expectedDateTime = new DateTime(2019, 11, 15, 13, 53, 26, 510);
+            var expectedGroupCourse = GroupTestHelper.GetDefaultGroupCourse(

```

Comment:
GWT is not traditionally how I have laid out unit tests. Arrange-Act-Assert has been the norm for me where everything is arranged up front.
Though I get what you mean, seeing the expected there at the end rather than having to scan back up to remind yourself. I can see there is currently a bit of a mix here. Given that we are defining GWT maybe we should set expectations at the end. Happy to go with that, and maybe something to write about in the swiki to keep the consistency.

Reply:
I'd thought GWT and AAA were synonymous - https://automationpanda.com/2020/07/07/arrange-act-assert-a-pattern-for-writing-good-tests/ has an example that puts the expected data in the Assert section. Guess there are just different ways of doing it.